background/background.go

```diff
@@ -76,9 +77,25 @@ func blockConnected() {
 	for _, ticket := range unconfirmed {
 		tktTx, err := dcrdClient.GetRawTransaction(ticket.Hash)
 		if err != nil {
-			log.Errorf("GetRawTransaction error: %v", err)
+			// Hitting this error probably indicates a tx which was never mined
+			// and has been removed from the mempool. For example, a ticket
+			// purchase tx close to an sdiff change, or a ticket purchase tx
+			// which expired. Remove it from the db.
+			if strings.Contains(err.Error(), "No information available about transaction") {
```

Comment:
Does this also work?
```
import "github.com/decred/dcrd/dcrjson/v3"
...
if errors.Is(err, dcrjson.ErrNoTxInfo) {
...
```

Reply:
I don't think this will work because `dcrjson.ErrNoTxInfo` is actually a different error. The one received by vspd has a different message which includes the transaction hash, and it also does not include the -5 code. There is some examples of the error in #128 

Its the same issue here https://github.com/decred/vspd/blob/master/rpc/dcrd.go#L105

dcrd does not include error codes in its rpc responses, just strings :(

Reply:
Oh. That's confusing. Thanks for pointing those lines out.

Reply:
I've just discovered that unlike `sendrawtransaction`, this RPC actually does return a proper code, so we can match on the code rather than the string. Updated the PR.